### ISSUE TITLE
Add class prop on Form component

### ISSUE THREAD

**Issue by gushonorato** [surface-ui/surface#320]
Surface let me pass classes into form inputs using a class property like this:

```
<TextInput class="lero" .../>
```

but when working with `Form` I must pass class using opts:

```
<Form opts={{ class: "lero" }} .../>
```

I think Surface should permit pass classes in forms using props, like we do in inputs. It would become the design more orthogonal.

**Comment by Malian** [comment]
Hi @gushonorato !

You are right, we should be consistent with other Form components.

Would you like to give a try to fix this? You can take a look at the LiveFileInput component for inspiration https://github.com/surface-ui/surface/blob/master/lib/surface/components/live_file_input.e

**Comment by msaraiva** [comment]
Fixed in #327.
